src/main/java/secondtask/NumberPi.java

```diff
@@ -0,0 +1,24 @@
+package secondtask;
+
+import utils.*;
+
+import java.io.FileNotFoundException;
+
+public class NumberPi {
+
+    private static final String INPUTFILE = "src/main/resources/input_files/input_secondtask.txt";
+
+    public static void main(String[] args) throws FileNotFoundException {
+        int n = InputOutputUtils.readOnlyOneIntFromFile(INPUTFILE);
+        double temp, pi = 0;
+        for (int i = 1; i <= Math.pow(10, n); i++) {
+            if (i % 2 == 0) {
+                temp = -4.0 / (2 * i - 1);
+            } else {
+                temp = 4.0 / (2 * i - 1);
+            }
+            pi += temp;
+        }
+        System.out.println("Pi equals to: " + DecimalUtils.roundDecimalNumber(pi, n));
```

Comment:
please remove the System.out.println(), the result should we written in the output file

Reply:
Modified as requested.
Also changed the method "writeToFile" in InputOutputUtils class.